docs/community/com-config/com-config-flags.md

```diff
@@ -9,64 +9,63 @@ grand_parent: Community
-| tls.key                        | PILOSA_TLS_KEY                        | FEATUREBASE_TLS_KEY                               | str   |      |
-| tls.skip-verify                | PILOSA_TLS_SKIP_VERIFY                | FEATUREBASE_TLS_SKIP_VERIFY                       | bool  |      |
-| tracing.agent-host-port        | PILOSA_TRACING_AGENT_HOST_PORT        | FEATUREBASE_TRACING_AGENT_HOST_PORT               | str   |      |
-| tracing.sampler-param          | PILOSA_TRACING_SAMPLER_PARAM          | FEATUREBASE_TRACING_SAMPLER_PARAM                 | float |      |
-| tracing.sampler-type           | PILOSA_TRACING_SAMPLER_TYPE           | FEATUREBASE_TRACING_SAMPLER_TYPE                  | str   |      |
-| translation.map-size           | PILOSA_TRANSLATION_MAP_SIZE           | FEATUREBASE_TRANSLATION_MAP_SIZE                  | int   |      |
-| translation.primary-url        | PILOSA_TRANSLATION_PRIMARY_URL        | FEATUREBASE_TRANSLATION_PRIMARY_URL               | str   |      |
-| auth.enable                    | PILOSA_AUTH_ENABLE                    | FEATUREBASE_AUTH_ENABLE                           | bool  |      |
-| auth.client-id                 | PILOSA_AUTH_CLIENT_ID                 | FEATUREBASE_AUTH_CLIENT_ID                        | str   |      |
-| auth.client-secret             | PILOSA_AUTH_CLIENT_SECRET             | FEATUREBASE_AUTH_CLIENT_SECRET                    | str   |      |
-| auth.authorize-url             | PILOSA_AUTH_AUTHORIZE_URL             | FEATUREBASE_AUTH_AUTHORIZE_URL                    | str   |      |
-| auth.token-url                 | PILOSA_AUTH_TOKEN_URL                 | FEATUREBASE_AUTH_TOKEN_URL                        | str   |      |
-| auth.group-endpoint-url        | PILOSA_AUTH_GROUP_ENDPOINT_URL        | FEATUREBASE_AUTH_GROUP_ENDPOINT_URL               | str   |      |
-| auth.redirect-base-url         | PILOSA_AUTH_REDIRECT_BASE_URL         | FEATUREBASE_AUTH_REDIRECT_BASE_URL                | str   |      |
-| auth.logout-url                | PILOSA_AUTH_LOGOUT_URL                | FEATUREBASE_AUTH_LOGOUT_URL                       | str   |      |
-| auth.scopes                    | PILOSA_AUTH_SCOPES                    | FEATUREBASE_AUTH_SCOPES                           | list  |      |
-| auth.secret-key                | PILOSA_AUTH_SECRET_KEY                | FEATUREBASE_AUTH_SECRET_KEY                       | str   |      |
-| auth.permissions               | PILOSA_AUTH_PERMISSIONS               | FEATUREBASE_AUTH_PERMISSIONS                      | str   |      |
-| auth.query-log-path            | PILOSA_AUTH_QUERY_LOG_PATH            | FEATUREBASE_AUTH_QUERY_LOG_PATH                   | str   |      |
-| auth.configured-ips            | PILOSA_AUTH_CONFIGURED_IPS            | FEATUREBASE_AUTH_CONFIGURED_IPS                   | str   |      |
+| CLI Flag                       | Environment Variable (with ['--future.rename'](#future-rename) flag) | Type  | Note                   |
```

Comment:
Note is not populating in table. likely due to large spacing here

This is taking up a tone of space:

> Environment Variable (with ['--future.rename'](#future-rename) flag)

Are we trying to move away from the old? If so, can our default have --future.rename and have this just be:

> Environment Variable

Reply:
Another idea. Make this a markdown warning that this is with the future.rename on

Reply:
Good idea! put future.rename into a warning, Also renamed "Note" to "Description" so it shows better in the table.